src/wallet/containers/WalletContainer.ts

```diff
@@ -9,13 +9,16 @@ const mapStateToProps = (state: SiteState) => {
     walletState: state.wallet.walletState,
     challengeState: state.wallet.challenge,
     showWallet: state.wallet.display.showWallet,
+    // TODO: We should store this in the wallet state and gert it from there
```

Comment:
```suggestion
    // TODO: We should store this in the wallet state and get it from there
```
Nit, misspelled comment